### ISSUE TITLE
KeyboardKey class>>#initializeKeyFromCharacterCompatibilityTable maps 3 characters incorrectly

### ISSUE THREAD

**Issue by koendehondt** [guillep/pharo-core#13150]
**Bug description**

`KeyboardKey class>>#initializeKeyFromCharacterCompatibilityTable` maps `Character` instances to key names.

```
initializeKeyFromCharacterCompatibilityTable
  KeyFromCharacterTable := Dictionary new.
  {
    Character backspace  . #Backspace "XK_BackSpace".
    Character tab        . #Tab "XK_Tab".
    Character cr         . #Return "XK_Return".
    Character escape     . #Escape "XK_Escape".
    Character home       . #Home "XK_Home".
    Character arrowLeft  . #Left "XK_Left".
    Character arrowUp    . #Up "XK_Up".
    Character arrowRight . #Right "XK_Right".
    Character arrowDown  . #Down "XK_Down".
    Character pageUp     . #Page_Up "XK_Page_Up".
    Character pageDown   . #Page_Down "XK_Page_Down".
    Character end        . #End "XK_End".
    Character space      . #Space "XK_Space".
    $/                   . #SLASH.
    $]                   . #BRACKETLEFT.
    $[                   . #BRACKETRIGHT.
    $(                   . #parenleft.
    $)                   . #parenright.
    $<                   . #less.
    $<                   . #greater.
    $|                   . #bar.
    $-                   . #minus}
      pairsDo: [ :character :keyname |
        KeyFromCharacterTable at: character put: (self named: keyname asUppercase) ]
```

Most of the mappings are correct, but these are wrong:
```
$]                  . #BRACKETLEFT.
$[                  . #BRACKETRIGHT.
$<                  . #greater.
```

**To Reproduce**

n/a

**Expected code**

```
$[                  . #BRACKETLEFT.
$]                  . #BRACKETRIGHT.
$>                  . #greater.
```

**Screenshots**

n/a

**Version information:**
 - OS: not relevant
 - Version: not relevant
 - Pharo Version: 9, 10, 11

**Expected development cost**
> As you know nothing happens magically, so do you have an idea of the development effort required? 
> In addition, are you willing to help because after all Pharo is yours too?

I guess the development effort is small.
Of course I like to help :-)

**Additional context**
None.

**Comment by Ducasse** [comment]
Thank koen. Do I read correctly that 
```
$]                  . #BRACKETLEFT.
$[                  . #BRACKETRIGHT.
```
should be 
```
$]                  . #BRACKETRIGHT.
$[                  . #BRACKETLEFT.
```

I found strange that we have 

```
    $<                   . #less.
    $<                   . #greater.
```

What should it be? less?

**Comment by koendehondt** [comment]
Yes, you read it correctly, because `[` is the left bracket and `]` is the right bracket. I noticed that the two were swapped when adding key combinations to a`SpCodePresenter` like this:
```
codePresenter
   bindKeyCombination: $] meta toAction: [ "do something" ];
   bindKeyCombination: $[ meta toAction: [ "do something else ]
```

The mapping of `$<` and `$>` does not interest me much, but I saw the strange code.
```
codePresenter bindKeyCombination: $> meta toAction: [ "do something" ]
```
raises an exception. When changing `KeyboardKey class>>#initializeKeyFromCharacterCompatibilityTable` so that it reads:
```
$> . #greater.
```
instead of:
```
$< . #greater.
```
and after sending`KeyboardKey initializeKeyFromCharacterCompatibilityTable`, the exception is gone.
